lib/embed/embed.dart

```diff
@@ -97,11 +97,11 @@ class PlaygroundMobile {
   }
 
   /**
-   * Return true if strong mode should be enabled
+   * Return true if strong mode should be enabled. Defaults to true.
    */
   bool _parseStrongModeParam(String strongModeValueString) {
-    if (strongModeValueString == null) return false;
-    return (strongModeValueString == 'true' || strongModeValueString == 't');
+    if (strongModeValueString == null) return true;
+    return !(strongModeValueString == 'f' || strongModeValueString == 'false');
```

Comment:
What about if `strongModeValueString == null || strongModeValueString.isEmpty` return true

otherwise try and parse it to a bool, and use the result (if it doesn't throw)

that would support `t` or `f`, but I don't think we need support the abbreviations

Reply:
Is there a specific parse method for bool that you have in mind? 
I don't think there's a default string to bool parser provided in the dart library.

Reply:
Hmm, not clear why, but I did think there was a bool parse method. It looks like there isn't; `strongModeValueString != 'false'` sgtm :)